Formula/textmate-emacs.rb

```diff
@@ -0,0 +1,25 @@
+require File.expand_path("../../Homebrew/emacs_formula", __FILE__)
+
+class TextmateEmacs < EmacsFormula
+  desc "Textmate-inspired minor mode"
+  homepage "https://github.com/defunkt/textmate.el"
+  url "https://github.com/defunkt/textmate.el/archive/v1.tar.gz"
+  sha256 "1ce4475d4e80eed145a5055f25912d66e93b4ca10306679b09e9cc722f5bcec2"
+  head "https://github.com/defunkt/textmate.el.git"
+
+  depends_on :emacs => "22.1"
```

Comment:
This was just from the template, right?  Could try dropping the version specification; I don't see anything obvious requiring a newish version.